README.md

```diff
@@ -29,15 +29,15 @@ npm install --save mapbox-gl @mapbox/mapbox-gl-language
 ```
 
 ```javascript
-var mapboxgl = require('mapbox-gl')
-var MapboxLanguage = require('@mapbox/mapbox-gl-language');
+const mapboxgl = require('mapbox-gl')
+const MapboxLanguage = require('@mapbox/mapbox-gl-language');
```

Comment:
Nit: should we use the ESM syntax for these examples instead of `require`?

Reply:
Good catch on the old code. Fixed.